### ISSUE TITLE
jquery-cookie.js not working

### ISSUE THREAD

**Issue by fafualex** [Irrelon/jquery-lang-js#57]
The changing language works perfectly but when I load another page the language returns to default.

**Comment by kevkha** [comment]
It works perfectly for me. Where and how do you define your jquery-cookie.js?

**Comment by fafualex** [comment]
I have put all in the head in this order:
 <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js" type="text/javascript"></script>
    <script src="assets/js/jquery-cookie.js" charset="utf-8" type="text/javascript"></script>
	<script src="assets/js/jquery-lang.js" charset="utf-8" type="text/javascript"></script>
	<script src="assets/js/langpack/nonDynamic.js" charset="utf-8" type="text/javascript"></script>
    <script type="text/javascript">
    // Create language switcher instance and set default language to it
        window.lang = new Lang('it');
    </script>

Thanks in advance  ;)

**Comment by kevkha** [comment]
I don't see your jquery-cookie.js line. Please check example from index.html of this project.

**Comment by fafualex** [comment]
I wrote it here.. Is the second script

**Comment by kevkha** [comment]
I see it now. Did you create your new translation file <code>it.json</code>? Comment out your <code>nonDynamic.js</code> line. Try like this with your it.json file
```html
<script src="assets/js/jquery-cookie.js" charset="utf-8" type="text/javascript"></script>
<script src="assets/js/jquery-lang.js" charset="utf-8" type="text/javascript"></script>
<script type="text/javascript">
	var lang = new Lang('it');
	lang.dynamic('it', 'assets/js/langpack/it.json');
</script>
```

**Comment by kevkha** [comment]
Btw, if you wish to insert HTML here, see http://stackoverflow.com/questions/21039126/how-to-use-html-code-in-github-issue-as-part-of-message

**Comment by fafualex** [comment]
Is not working like this.. neither the translation works.. 
I'll explain my situation: I have a website in Italian set as primary language, and with nonDynamic.js works perfectly.
This is my nonDynamic.js:
```
Lang.prototype.pack.en = {
	"token": {
		"L'azienda":"The winery",
		"I vini":"Wines"
         },
};

**Comment by Irrelon** [comment]
Have you checked your browser to see if a cookie gets created when you switch languages?

**Comment by fafualex** [comment]
No cookies gets created when I change language.. I have other cookies from the page but not the language one..

**Comment by kevkha** [comment]
I tested your ways and it works fine for me too. I have a cookie. Are you testing cookie with local files? You need a web server or make your browser to startup with cookie option depending on the browser you use.

<code>nonDynamic.js</code>
```javascript
Lang.prototype.pack.en = {
    "token": {
        "L'azienda":"The winery",
        "I vini":"Wines"
        },
};
```
<code>index.html</code>
```html
<!DOCTYPE html>
<html>
<head>
	<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
	<title>jQuery Lang JS Module Test Page</title>
	<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js" type="text/javascript"></script>
	<script src="js/jquery-cookie.js" charset="utf-8" type="text/javascript"></script>
	<script src="js/jquery-lang.js" charset="utf-8" type="text/javascript"></script>
	<script src="js/langpack/nonDynamic.js" charset="utf-8" type="text/javascript"></script>
	<script type="text/javascript">
		var lang = new Lang('it');
	</script>
	<style>
		body {
			font-family: Verdana, Geneva, sans-serif;
			font-size: 12px;
		}

		#langChanger {
			margin-bottom: 10px;
		}

		.sectionTitle {
			font-weight: bold;
			font-size: 14px;
			margin-bottom: 10px;
		}
	</style>
</head>

<body>
<div id="langChanger"><a href="#" onclick="window.lang.change('it'); return false;">Switch to IT</a> | <a href="#" onclick="window.lang.change('en'); return false;">Switch
	to EN</a></div>
<div id="testHtml">
	<h1 lang="it">L'azienda<h1>
	<h2 lang="it">I vini<h2>
	<a href="index2.html">Page 2</a>
</div>
</body>
</html>
```
<code>index2.html</code>
```html
<!DOCTYPE html>
<html>
<head>
	<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
	<title>jQuery Lang JS Module Test Page</title>
	<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js" type="text/javascript"></script>
	<script src="js/jquery-cookie.js" charset="utf-8" type="text/javascript"></script>
	<script src="js/jquery-lang.js" charset="utf-8" type="text/javascript"></script>
	<script src="js/langpack/nonDynamic.js" charset="utf-8" type="text/javascript"></script>
	<script type="text/javascript">
		var lang = new Lang('it');
	</script>
	<style>
		body {
			font-family: Verdana, Geneva, sans-serif;
			font-size: 12px;
		}

		#langChanger {
			margin-bottom: 10px;
		}

		.sectionTitle {
			font-weight: bold;
			font-size: 14px;
			margin-bottom: 10px;
		}
	</style>
</head>

<body>
<div id="langChanger"><a href="#" onclick="window.lang.change('it'); return false;">Switch to IT</a> | <a href="#" onclick="window.lang.change('en'); return false;">Switch
	to EN</a></div>
<div id="testHtml">
	<h1 lang="it">L'azienda<h1>
	<h2 lang="it">I vini<h2>
	<a href="index.html">Page 1</a>
</div>
</body>
</html>
```
![image](https://cloud.githubusercontent.com/assets/1807308/11218539/2d2a77a6-8d1b-11e5-9746-1de0379860d2.png)

![image](https://cloud.githubusercontent.com/assets/1807308/11218570/5ae2efc0-8d1b-11e5-8493-df72585c6a64.png)

**Comment by fafualex** [comment]
Thank you for the exhaustive answer!! I'll try to see if I've miss something and let you know.. ;) really appreciate that!

**Comment by kevkha** [comment]
@fafualex: FYI, @Irrelon merged my pull request #59 to master branch. You might want to check that out. Good lucks!
